.github/workflows/code-checks.yml

```diff
@@ -34,7 +34,8 @@ jobs:
       run: |
         pip install torch==1.11 --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
         python ./requirements/pytorch/adjust-versions.py requirements/pytorch/extra.txt
-        pip install -r requirements/pytorch/devel.txt --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
+        pip install -e . --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
```

Comment:
I think this change should be excluded from this PR as it changes the mypy configuration, hence the mypy failures.

It can be a follow-up

Reply:
```suggestion
        # todo: adjust requirements for both code-bases
        pip install -r requirements/pytorch/devel.txt --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
```